packages/process/src/node/process.ts

```diff
@@ -29,9 +29,11 @@ export abstract class Process {
     protected _killed = false;
 
     constructor(
-        @inject(ProcessManager) protected readonly processManager: ProcessManager,
+        protected readonly processManager: ProcessManager,
         protected readonly logger: ILogger,
-        readonly type: ProcessType) {
+        @unmanaged() readonly type: ProcessType,
+        @unmanaged() protected readonly command: string,
```

Comment:
I'd rather we pass the options object

Reply:
Actually they are the same, I would like to keep two different structs however so that they can differ in the future but we should make them come from a base ProcessOptions and use that.